### ISSUE TITLE
Masking not done correctly in VQ?

### ISSUE THREAD

**Issue by harish-kamath** [collabora/WhisperSpeech#62]
https://github.com/collabora/WhisperSpeech/blob/b6fc87c9e1ffd28b0fef71bc25896ce323a76fa8/whisperspeech/vq_stoks.py#L88

Shouldn't this mask the actual padding area, which includes a random part of both the left and right part of the audio?

**Comment by jpc** [comment]
Yes, you're right. In the code you pointed out we assume padding is always from the right and the rest of the code allows for randomized padding. I think it did not cause issues in practice because I never enabled the random shift option here:

https://github.com/collabora/WhisperSpeech/blob/b6fc87c9e1ffd28b0fef71bc25896ce323a76fa8/whisperspeech/wh_transcribe.py#L93
